configure.ac

```
@@ -14,9 +14,10 @@ AC_ARG_ENABLE(contrib,
 
 AC_ARG_ENABLE(real,
 	      AS_HELP_STRING([--enable-real=Xp  Specify working precision of REAL types:
-                    sp -- REAL(kind=4)
-	            dp -- REAL(kind=8) (default)
-	            qp -- REAL(kind=16)]),
+                    ssp -- REAL(kind=4) xp -- sp 
+                    sp -- REAL(kind=4) xp -- dp  
+	            dp -- REAL(kind=8) xp -- dp (default)
+	            qp -- REAL(kind=16) xp -- qp]),
```

Inconsistent whitespace.
```suggestion
                    dp -- REAL(kind=8) xp -- dp (default)
                    qp -- REAL(kind=16) xp -- qp]),
```